addons/point_of_sale/static/src/app/models/pos_order.js

```diff
@@ -947,7 +947,11 @@ export class PosOrder extends Base {
     /* ---- Ship later --- */
     //FIXME remove this
     setShippingDate(shippingDate) {
-        this.shipping_date = shippingDate;
+        if (shippingDate) {
+            this.shipping_date = serializeDateTime(deserializeDate(shippingDate));
+        } else {
+            this.shipping_date = shippingDate;
+        }
```

Comment:
We should verify where this method is called to be sure we always give UTC DT  or local DT. If not the result will not be correct